test/runner.js

```diff
@@ -0,0 +1,185 @@
+'use strict'
+
+let fs = require('fs')
+let path = require('path')
+let expect = require('chai').expect
+let rimraf = require('rimraf')
+let Launcher = require('../node_modules/webdriverio/build/lib/launcher.js')
```

Comment:
Never use `node_modules` directly in require statements.

Here you can just use this:
```js
require('webdriverio/build/lib/launcher');
```